orange_setup/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: copy rpms to server
```

Comment:
I don't think we should have the RPMs as artefacts in this repo, but how we handle them otherwise is TBD.  Best option may be to make a locally-accessible repo server which contains the OrangeFS RPMs, plus a few others we've got kicking around.